department-of-veterans-affairs/va.gov-cms
# VAGOV-5520: Adding comments to local patches.

## What this does
 - Adds comments to local patches explaining that they are not suitable for contrib, as they are project specific.

## QA
 - Visit /patches, and confirm that all three files have a comment above diff explaining why they are local.
